Sharpmake.Generators/VisualStudio/Vcxproj.cs

```diff
@@ -824,7 +824,9 @@ Strings ignoreSpecificLibraryNames
 
                 if (extension != platformVcxproj.StaticLibraryFileFullExtension && extension != platformVcxproj.SharedLibraryFileFullExtension)
                 {
-                    decoratedName = libPrefix + libraryFile;
+                    string dir = Path.GetDirectoryName(libraryFile);
+                    string file = Path.GetFileName(libraryFile);
+                    decoratedName = Path.Combine(dir, libPrefix + file);
```

Comment:
same here

Reply:
Hrm, testing it now I don't seem to hit this code anymore so I guess I can leave it out for now until I get it again and revisit it.